test/Twig/Tests/Node/ForTest.php

```diff
@@ -73,13 +73,13 @@ public function getTests()
 
         $tests[] = array($node, <<<EOF
 // line 1
-\$context['_parent'] = (array) \$context;
-\$context['_seq'] = twig_ensure_traversable({$this->getVariableGetter('values')});
-\$context['loop'] = array(
-  'parent' => \$context['_parent'],
-  'index0' => 0,
-  'index'  => 1,
-  'first'  => true,
+\$context["_parent"] = (array) \$context;
```

Comment:
semantically we want to keep the single quotes in these places. double quotes are only used when around variables.